src/patchelf.cc

```diff
@@ -1057,11 +1057,11 @@ void ElfFile<ElfFileParamNames>::rewriteHeaders(Elf_Addr phdrAddress)
 
     /* If there is a segment for the program header table, update it.
        (According to the ELF spec, there can only be one.) */
-    for (auto phdr : phdrs) {
```

Comment:
This can be fixed if you change to auto& , no?

Reply:
yes, "auto &" also can fixed it.

Reply:
need I  send a new pull request?